Wikipedia/Code/WMFWelcomeAnalyticsViewController.m

```diff
@@ -42,6 +44,22 @@ - (void)viewDidLoad {
     [self wmf_setupTransparentWelcomeNavigationBarWithBackChevron];
 }
 
+- (void)viewWillAppear:(BOOL)animated {
+    [super viewWillAppear:animated];
+    static dispatch_once_t onceToken;
```

Comment:
static means "For **ALL** instances for the entire life of the application."

I think what you mean is  "For **THIS** instance for its entire life."

Usually that means an ivar as opposed to a static variable.